library/cache-library/jcs-cache-service/src/main/resources/cache.ccf

```diff
@@ -16,6 +16,9 @@ jcs.auxiliary.DC.attributes.OptimizeOnShutdown=true
 # Region for named operations
 jcs.region.namedOperationsRegion=DC
 
+# Region for named views
+jcs.region.namedViewRegion=DC
```

Comment:
Please could you add a migration step to the issue to tell users that if they want to use NamedViews they will need to add this to any custom cache.ccf files they have. Also, for backwards compatibility please can you make sure you that if a user doesn't set this, then the app can handle it and doesn't crash.